### PR TITLE
CORE-923: Initial ETH Handlers

### PR DESCRIPTION
**Pull request by EBGToo**

### CODE REVIEW

File: WalletKitCore/src/crypto/handlers/eth/BRCryptoTransferETH.c [hunk 1]
```diff
@@ -1,59 +1,33 @@
...
+                             BREthereumAddress target);
 
-static BRCryptoTransferETH
+extern BRCryptoTransferETH
 cryptoTransferCoerce (BRCryptoTransfer transfer) {
     assert (CRYPTO_NETWORK_TYPE_ETH == transfer->type);
     return (BRCryptoTransferETH) transfer;
 }
 
+#if 0
```
Comment (ewerx) [inline WalletKitCore/src/crypto/handlers/eth/BRCryptoTransferETH.c:19]: still needed for reference?

File: WalletKitCore/src/crypto/handlers/eth/BRCryptoTransferETH.c [hunk 2]
```diff
@@ -185,11 +254,110 @@ cryptoTransferHasETH (BRCryptoTransfer transferBase,
...
+
+    BREthereumHash h1 = cryptoTransferGetEthHash (tb1);
+    BREthereumHash h2 = cryptoTransferGetEthHash (tb2);
 
-    return t1->tid == t2->tid;
+    return (ETHEREUM_BOOLEAN_FALSE != ethHashEqual (h1, EMPTY_HASH_INIT) &&
+            ETHEREUM_BOOLEAN_TRUE  == ethHashEqual (h1, h2));
 }
 
+#if 0
```
Comment (ewerx) [inline WalletKitCore/src/crypto/handlers/eth/BRCryptoTransferETH.c:270]: implemented in BRCryptoWalletManagerETH.c, can be removed?

File: WalletKitCore/src/crypto/handlers/eth/BRCryptoTransferETH.c (hunk 1 above)
Comment (EBGToo) [reply WalletKitCore/src/crypto/handlers/eth/BRCryptoTransferETH.c:19]: Let me leave this one there for now - for reference - given the size of the comments.

File: WalletKitCore/src/crypto/handlers/eth/BRCryptoTransferETH.c (hunk 2 above)
Comment (EBGToo) [reply WalletKitCore/src/crypto/handlers/eth/BRCryptoTransferETH.c:254]: This one I'll axe.

The subsequent one for `transferSign*` will disappear on soon-to-be followup PR.

File: WalletKitCore/src/crypto/handlers/eth/BRCryptoWalletETH.c [hunk 3]
```diff
@@ -1,144 +1,185 @@
...
+        case TRANSFER_BASIS_TRANSACTION:
+            return ethEtherCreate(value);
+        case TRANSFER_BASIS_LOG:
+            return ethEtherCreateZero ();
+    }
 }
 
-static BRCryptoTransfer
-cryptoWalletCreateTransferETH (BRCryptoWallet  wallet,
+#if 0
```
Comment (ewerx) [inline WalletKitCore/src/crypto/handlers/eth/BRCryptoWalletETH.c:117]: implemented below

**Pull request merged**